package.json

```diff
@@ -102,7 +102,8 @@
     "@apidevtools/json-schema-ref-parser": "9.0.9",
     "@types/node": "16.9.1",
     "@types/react": "18.0.26",
-    "@types/react-dom": "^18.0.9"
+    "@types/react-dom": "^18.0.9",
+    "zod": "^3.20.2"
```

Comment:
@zomars was this a mistake by me merging?

Reply:
Zod isn't needed on root AFAIK.